data-service/src/main/java/uk/gov/gchq/palisade/service/data/config/StdSerialiserPrepopulationFactory.java

```diff
@@ -99,11 +99,11 @@ public void setSerialiserClass(final String serialiserClass) {
         LOGGER.info("Building serialiser for class '{}', resource format '{}' and domain type '{}'", serialiserClass, flavourFormat, flavourType);
         try {
             Class<?> serialiserClazz = Class.forName(serialiserClass);
-            LOGGER.debug("Got class {} for serialiser classname {}", serialiserClazz, serialiserClass);
+            LOGGER.trace("Got class {} for serialiser classname {}", serialiserClazz, serialiserClass);
             Constructor<?> serialiserConstructor = serialiserClazz.getConstructor(Class.class);
-            LOGGER.debug("Got one-element constructor {} for class {}", serialiserConstructor, serialiserClazz);
+            LOGGER.trace("Got one-element constructor {} for class {}", serialiserConstructor, serialiserClazz);
             Class<?> typeClass = Class.forName(flavourType);
-            LOGGER.debug("Got class {} for flavourType classname {}", typeClass, flavourType);
+            LOGGER.trace("Got class {} for flavourType classname {}", typeClass, flavourType);
```

Comment:
Still wanted at trace?

Reply:
Yeah, i think it's all a bit too noisy for debug